package.json

```diff
@@ -2,13 +2,23 @@
   "name": "@tscircuit/3d-viewer",
   "version": "0.0.114",
   "main": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./3d": {
+      "types": "./dist/three-components/convert-3d-circuit-to-svg.d.ts",
+      "import": "./dist/three-components/convert-3d-circuit-to-svg.js"
+    }
+  },
```

Comment:
there's no need to separate exports anymore because there is no longer a peer dep